src/Aspire.Hosting/build/Aspire.Hosting.targets

```diff
@@ -63,13 +63,51 @@ public class ]]>%(ClassName)<![CDATA[ : IServiceMetadata
     </ItemGroup>
   </Target>
 
+  <Target Name="CreateProjectMetadata">
```

Comment:
@baronfel can you give me an after the fact review on this?

Reply:
This all seems ok to me, seems straightforward enough based on the existing patterns.